nservicebus/azure-service-bus/configuration/configuration.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Full Azure Service Bus Configuration API
+summary: List of all Azure Service Bus Transport configuration settings
+tags:
+- Azure
+- Cloud
+- Configuration
+---
+
+## Full Configuration API
+
+The full configuration API can be accessed from the `UseTransport<AzureServiceBusTransport>()` extension method and provides low level access to almost all aspects of the transport's behavior.
+ 
+## Configuring The Topology
+ 
+A topology defines what the underlying layout of Azure Service Bus messaging entities looks like, which ones are used and how they relate to each other. There are 2 topologies built in into the transport `EndpointOrientedTopology` and `ForwardingTopology`.
+
+ * `UseTopology<T>()`: Specifies which topology should be used by the transport, instantiating it using the default constructor.
+ * `UseTopology<T>(Func<T>)`: Specifies which topology should be used by the transport, instantiating it using a factory method.
+ * `UseTopology<T>(T)`: Specifies which topology should be used by the transport, instance created by the calling code.
+ 
```

Comment:
Link here to the topologies article, could be:
- For more information refer to the [Topologies](/nservicebus/azure-service-bus/topologies/) article
- Link the section names below to the appropriate sections, e.g.
> ### Endpoint Oriented Topology

### [Endpoint Oriented Topology](/nservicebus/azure-service-bus/topologies/#version-7-and-above-endpoint-oriented-topology)